components/pipeline-service/base/rbac/cluster-role/pipeline-service-sre.yaml

```diff
@@ -58,6 +58,16 @@ rules:
       - patch
       - update
       - watch
+  # This rule is needed to be able to diagnose Cluster Health. A diagnosis can be made in situations like
+  # PipelineRun Execution/Scheduling Overhead alert starts firing.
+  - apiGroups:
+      - ""
+    resources:
+      - "nodes"
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
ok running `oc adm top nodes --loglevel=10` @ramessesii2 after the api query I see these 2 gets:
```
I1108 09:11:20.075500   46504 round_trippers.go:553] GET https://api.gmontero412.devcluster.openshift.com:6443/apis/metrics.k8s.io/v1beta1/nodes 200 OK in 70 milliseconds
I1108 09:11:20.509320   46504 round_trippers.go:553] GET https://api.gmontero412.devcluster.openshift.com:6443/api/v1/nodes 200 OK in 68 milliseconds
```

your update here handles the second one.  But for the first, you'll need an additional stanza, where you set apiGroups to `metrics.k8s.io` and the resources again to `nodes` with the same verbs you have here.

Reply:
thanks for reviewing. Added your suggestion.